lessonbuilder/tool/src/webapp/js/show-page.js

```diff
@@ -1435,7 +1435,8 @@ $(document).ready(function() {
 				
 				row.find(".questionMultipleChoiceAnswer").each(function(index, el) {
 					var id = $(el).find(".questionMultipleChoiceAnswerId").text();
-					var text = $(el).find(".questionMultipleChoiceAnswerText").text();
+					//SAK-46296
+					var text = $(el).find(".raw-questionAnswer-text").val();
```

Comment:
```suggestion
					const text = el.querySelector(".raw-questionAnswer-text").value;
```

Reply:
I'd actually update this entire each function to just use vanilla js and stop wrapping the element in a jquery object, or at least just wrap it once. It's really inefficient. Just what's inside the each function though, so the change isn't too big. That way we steadily improve the js and move away from unnecessarily using jquery. It's up to you though, but that's what I'd do.

Reply:
Thank you for your input and I see your point. But this PR is intended to just fix the issue and in my opinion refactoring should be separate. So i would like on my side to keep the already used jquery.

Best regards.